src/btree/bt_sync_obsolete.c

```diff
@@ -506,6 +510,11 @@ __checkpoint_cleanup_walk_btree(WT_SESSION_IMPL *session, WT_ITEM *uri)
     }
 
     btree = S2BT(session);
+
+    /* Skip read-only btrees. */
+    if (F_ISSET(btree, WT_BTREE_READONLY))
```

Comment:
Should this be backported?

Reply:
Is it causing any problem?

Reply:
Not that I am aware of but looks like something we forgot.

Reply:
Feel free to ask for backport if this is needed.